docs/Data_Portal/Users_Guide/gene_expression_clustering.md

```diff
@@ -12,7 +12,7 @@ At the Analysis Center, click the 'Gene Expression Clustering' card to launch th
 
 Users can view publicly available genes as well as login with credentials to access controlled data.
 
-There are four main panels in the Gene Expression Clustering tool: controls, heatmap, variables, and legend.
+There are four main panels in the Gene Expression Clustering tool: [controls], [heatmap], [variables], and [legend].
```

Comment:
Not sure if you intended it look like this:


<img width="829" alt="Screenshot 2025-06-27 at 1 24 01 PM" src="https://github.com/user-attachments/assets/79de4c48-4d1b-48c1-ae63-b75371d25cc2" />